test/integration/visitor_can_view_deal_page_test.rb

```diff
@@ -2,17 +2,17 @@
 
 class VisitorCanViewDealPageTest < ActionDispatch::IntegrationTest
   include CategoryItemsSetup
-  test "visitor can view the daily deal page" do
+  test 'visitor can view the daily deal page' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.